src/Providers/IAnnualReportProvider.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Ageras\Sherlock\Providers;
+
+interface IAnnualReportProvider
```

Comment:
Please follow the standard of `CompanyProviderInterface`. This should be named `AnnualReportProviderInterface`